scripts/build.tokens.ckerc20.mjs

```diff
@@ -78,6 +79,37 @@ const ORCHESTRATOR_PRODUCTION_ID = Principal.fromText('vxkom-oyaaa-aaaar-qafda-c
 
 const DATA_FOLDER = join(process.cwd(), 'src', 'frontend', 'src', 'env');
 
+const LOGO_FOLDER = join(process.cwd(), 'src', 'frontend', 'src', 'icp-eth', 'assets');
+
+const saveTokenLogo = async (canisterId, name) => {
+	const logoName = name.toLowerCase().replace('ck', '').replace('sepolia', '');
+	const file = join(LOGO_FOLDER, `${logoName}.svg`);
+
+	if (existsSync(file)) {
+		return;
+	}
+
+	const agent = await createAgent({
+		identity: new AnonymousIdentity(),
+		host: 'https://icp-api.io'
+	});
+
+	const { metadata } = IcrcLedgerCanister.create({
+		agent,
+		canisterId
+	});
+
+	const data = await metadata({});
```

Comment:
It's the default but, to make it explicit I would specify `{ certified: true }` to express the fact that we make an update call. WDYT?

Reply:
done!